src/app.js

```diff
@@ -144,96 +250,63 @@ async function getFinalizedBlockNumber(api) {
   const header = await api.rpc.chain.getHeader(hash);
   return header.number;
 }
-  
-async function makeTransaction(api, sender, receiver, funds, timeout_ms)  {
-  const sleep_ms = 50;
-  let timed_out = false;
-  let transaction_error = null;
-  let hash = null;
-  // Verbose transaction information -- could be removed in the future
-  let nonce = await api.query.system.accountNonce(sender.address);
-  let sender_balance = await api.query.balances.freeBalance(sender.address);
-  let receiver_balance = await api.query.balances.freeBalance(receiver.address);
-  console.log(
-    `${sender.meta.name} [${sender_balance}] =>`, 
-    `${receiver.meta.name} [${receiver_balance}]`,
-    `: Nonce - ${nonce.words}`
-    );
-
-  // Sign and send a balance transfer
-  const unsub = await api.tx.balances.transfer(receiver.address, funds)
-  .signAndSend(sender, {nonce: nonce}, (result) => {
-    console.log(`Current status is ${result.status}`);
-    
-    if (result.isCompleted) {
-      if (result.isFinalized) {
-        hash = result.status.asFinalized;
-      }
-      else { // error
-        console.log("Transaction Rejected");
-        throw [APP_FAIL_TRANSACTION_REJECTED, `Transaction rejected ${result.status}`];
-      }
-      unsub();
-    }
-  })
-  .catch(err => transaction_error = err);
-  
-  const timer = setTimeout(() => timed_out = true, timeout_ms);
-  
-  // periodically checks whether we have timed out
-  // avoided sleeping here so that we can report a successful transation at the time
-  // it occurs.
-  while(timed_out == false) {
-    if (hash != null) {
-      clearTimeout(timer);
-      break;
-    }
-    if (transaction_error != null) {
-      throw [APP_FAIL_TRANSACTION_REJECTED, `Transaction rejected`];
-    }
-    await sleep(sleep_ms);
-  }
-  
-  return hash;
-}
 
 /// Generates a number of steve keypairs
 function createTheSteves(number, steve_keyring) {
```

Comment:
Lets move these steves out of this module too - the create and fund steves don't need to be in this file

Reply:
Agree, moving them to the util